src/WorkerChannel.ts

```diff
@@ -83,6 +83,7 @@ export class WorkerChannel implements IWorkerChannel {
    */
   public workerInitRequest(requestId: string, msg: rpc.WorkerInitRequest) {
     let capabilitiesDictionary = {};
+    capabilitiesDictionary['RpcTriggerMetadataPopulated'] = "true";
     capabilitiesDictionary['RpcHttpBodyOnly'] = "true";

```

Comment:
You can also define it as:
```javascript
const capabilitiesDictionary = {
     RpcTriggerMetadataPopulated: "true"
     RpcHttpBodyOnly: "true"
};
```

Slight preference for this, as the capabilities dictionary should be a constant

Reply:
Also name is a little confusing (commented on other PR too!) can it be something like `RemoveRpcHttpTriggerMetadata`